File: Module1/8-8/HierarchicalEx.java
package com.a88;

class Bank
{
	void bank()
	{
		System.out.println("banking");
	}
}
class Current extends Bank
{
	void current()
	{
		System.out.println("current");
	}
}
class Save extends Bank
{
	void save()
	{
		System.out.println("Saving");
	}
}

public class HierarchicalEx 
{
	
	public static void main(String[] args) 
	{
		Current c =new Current();
		Save s =new Save();
		
		c.current();
		s.save();
		s.bank();
	}
	
}

File: Module1/Kotlin/26-8/HierarchicalEx.kt
open class Bank
{
    fun bank()
    {
        println("Banking")
    }
}
class Current :Bank()
{
    fun current()
    {
        println("Current detail accessed")
    }

}
class Save :Bank()
{
    fun save()
    {
        println("saving")
    }
}
fun main()
{
    var c=Current()
    var s=Save()

    c.current()
    s.save()
    c.bank()
}